data.sql

```diff
@@ -84,7 +84,14 @@ INSERT INTO visits (animal_id, vet_id, date) VALUES (1,1,'2020-05-24'),
 
 
 
-
+INSERT INTO visits (animal_id, vet_id, date)
+SELECT animal_ids.id, vets_ids.id, date_trunc('day', visit_timestamp)
+FROM (SELECT id FROM animals) animal_ids,
+     (SELECT id FROM vets) vets_ids,
+     generate_series('1980-01-01'::timestamp, '2021-01-01', '4 hours') AS visit_timestamp;
+
+     INSERT INTO owners (full_name, email)
+SELECT 'Owner ' || generate_series(1, 2500000), 'owner_' || generate_series(1, 2500000) || '@mail.com';
```

Comment:
- [ ] As the requirement for this project, you should screenshots of **_before_** and **_after_** each problematic query. Currently, the screenshots you have attached are incomplete or very difficult to read. Please see the image below to see how to present the screenshots that will be easily deciphered for each. 

![image](https://user-images.githubusercontent.com/84156714/216977209-80a33701-df1b-46e0-9860-583e449d5a8a.png)


I see you are taking screenshots of your full screen, but you can use the snipping tool instead to take screenshots of specific parts of the screen. Learn how to use it [here](https://support.microsoft.com/en-us/windows/open-snipping-tool-and-take-a-screenshot-a35ac9ff-4a58-24c9-3253-f12bac9f9d44)

Please replace those screenshots in your pull request before you request another review. The next reviewer will check the screenshot to see if the performance issues have been fixed. Goodluck